### PR TITLE
nmcli: Type: Wrong package names

### PR DESCRIPTION
**Pull request by fbruetting**
In Red Hat systems, python packages are preceeded by `python3-`

##### ISSUE TYPE

- Bugfix Pull Request
- Docs Pull Request
- Feature Pull Request
- New Module Pull Request

##### ADDITIONAL INFORMATION

```paste below

```

### CODE REVIEW

**Comment by felixfontein** [comment]
Are you sure that all the systems mentioned use Python 3? I'm pretty sure that packages for Python 2 are not prefixed by `python3-`.

**Comment by fbruetting** [comment]
Python 2 is EOL, isn't it? In current Fedora/CentOS/RHEL there is no more `python` - and even if it would be, the package name would be `python-x` instead of `x-python`.

So:

* The current wording doesn't work under Red Hat systems at all (neither old nor new python), whereas it would work for all current systems (except for maybe RHEL/CentOS 6) with `python3-x`. Either way it's an improvement.

* I think we can consider python2 a special case, which also nobody should encounter anymore (and if they do, they hopefully know what they're doing).

**Comment by felixfontein** [comment]
Python 2 is EOL, but that doesn't mean it's not used anymore :) I'm not using any RH OSes, so I don't know which Python versions are still in use where, but I know for a fact that Ansible still even supports Python 2.6 (!) for modules, and I don't think they would if nobody is using it anymore.

**Comment by fbruetting** [comment]
As said, the current wording doesn't work anywhere, so the change would at least cause less confusion for the vast majority, as they won't have to find out why the package name cannot be found and have to search for the correct names.

For people using unsafe python, nothing would change, as both versions won't work. 🤷‍♂️

**Comment by felixfontein** [comment]
I checked some older Ansible CI runs, CentOS 7 and Fedora 28 both seem to be using Python 2.7 by default (Fedora 29 and CentOS 8 use Python 3.x).

**Comment by fbruetting** [comment]
Then we need `python-x` for those and `python3-x` for the current versions. I'll update the PR later.

**Comment by fbruetting** [comment]
In Fedora, Python 3 is the default since version 31[source](https://fedoraproject.org/wiki/Changes/Python_means_Python3), older Fedora versions are EOL [source](https://en.wikipedia.org/wiki/Fedora_(operating_system)), so I won’t respect these. This just leaves CentOS/RHEL 6/7 using Python 2.

How can I stash these two commits? Or would they get stashed at merge time? (I set the “allow edits by maintainers”)

File: plugins/modules/net_tools/nmcli.py [hunk 1]
```diff
@@ -20,10 +20,10 @@
 - nmcli
 description:
     - Manage the network devices. Create, modify and manage various connection and device type e.g., ethernet, teams, bonds, vlans etc.
-    - 'On CentOS 8 and Fedora >=29 like systems, the requirements can be met by installing the following packages: NetworkManager-libnm,
-      libsemanage-python, policycoreutils-python.'
-    - 'On CentOS 7 and Fedora <=28 like systems, the requirements can be met by installing the following packages: NetworkManager-glib,
-      libnm-qt-devel.x86_64, nm-connection-editor.x86_64, libsemanage-python, policycoreutils-python.'
+    - 'On CentOS/RHEL 8 and Fedora systems, the requirements can be met by installing the following packages: NetworkManager-libnm,
```
Comment (felixfontein) [inline plugins/modules/net_tools/nmcli.py:23]: Why did you remove the information on Fedora versions?

Comment (fbruetting) [reply plugins/modules/net_tools/nmcli.py:23]: All versions with python2 as default are not supported anymore.

Comment (felixfontein) [reply plugins/modules/net_tools/nmcli.py:23]: So what? Why remove information that is potentially valuable just because something is not supported anymore?

**Comment by Andersson007** [comment]
IMO, would be good just to add info related to latest CentOS/RHEL/FEDORA without changing existing stuff.
And the best place for such information is `notes:` subsection, isn't it? Definitely not the description, looks a bit noisy now.
So i'd move this stuff to `notes:`. What do you folks think about it?

File: plugins/modules/net_tools/nmcli.py (hunk 1 above)
Comment (Akasurde) [reply plugins/modules/net_tools/nmcli.py:23]: Please keep older information as well as people might require it.

**Comment by fbruetting** [comment]
Even Fedora 30 is EOL - why do you want to retain information about Fedora 28, which went EOL two years ago? That's superfluous, and just complicates things.

**Comment by felixfontein** [comment]
Even if something is EOL, that does not mean that nobody is using it anymore. (Would be nice, but unfortunately that's just not the case.) So this information is still useful for some people.

**Comment by Andersson007** [comment]
@fbruetting any updates on this?

**Comment by felixfontein** [comment]
Closing in favor of #712.
